pyface/fields/tests/test_text_field.py

```diff
@@ -1,87 +1,70 @@
-from __future__ import absolute_import
+# Copyright (c) 2019, Enthought, Inc.
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
 
 from traits.testing.unittest_tools import unittest
 
-from pyface.action.api import Action, MenuManager
+from pyface.toolkit import toolkit
 from ..text_field import TextField
+from .field_mixin import FieldMixin
 
 
-class TestTextField(unittest.TestCase):
+is_wx = (toolkit.toolkit == 'wx')
 
-    def test_text_field(self):
-        widget = TextField(
-            parent=None,
-            value='test',
-            tooltip='Dummy',
-        )
 
-        widget._create()
-        try:
-            widget.show(True)
-            widget.value = 'new value'
+class TestTextField(unittest.TestCase, FieldMixin):
 
-            self.assertEqual(widget._get_control_value(), 'new value')
-        finally:
-            widget.destroy()
+    def setUp(self):
+        FieldMixin.setUp(self)
```

Comment:
Is there some reason to avoid calling `unittest.TestCase.setup()`?

Reply:
Yes, see above.